keyboards/lazydesigners/dimple/dimple.c

```diff
@@ -22,3 +22,8 @@ void dimple_led_on() {
 void dimple_led_off() {
   DDRE &= ~(1 << 6); PORTE &= ~(1 << 6);
 }
+
+void keyboard_post_init_kb(void) {
```

Comment:
Honestly, I think that `matrix_init_*` or `keyboard_pre_init_*` would be a better place to  call this.